tests/integration/data/test_data_a11y.json

```diff
@@ -0,0 +1,63 @@
+{
+    "zones": [
+        {
+            "index": 1,
+            "title": "Zone 1",
+            "description": "This describes zone 1",
```

Comment:
@bradenmacdonald Using any one of the existing fixtures breaks the tests in `test_render.py` (before/without the changes that this PR introduces) since they are relying on a different configuration of items and zones than the tests that use existing fixtures.